diff --git a/.reviewboardrc b/.reviewboardrc
new file mode 100644
index 0000000..1446cb9
--- /dev/null
+++ b/.reviewboardrc
@@ -0,0 +1,7 @@
+REVIEWBOARD_URL = "192.168.232.158"
+REPOSITORY = "peihe"
+REPOSITORY_TYPE = "git"
+BRANCH = "master"
+LAND_DEST_BRANCH = "master"
+USERNAME="lipeihe"
+PASSWORD="123456"
diff --git a/t1.txt b/t1.txt
new file mode 100644
index 0000000..4dadf16
--- /dev/null
+++ b/t1.txt
@@ -0,0 +1,2 @@
+dddddddddd
+eeeeeeeeeee
